src/components/AboutModal.tsx

```diff
@@ -115,7 +115,9 @@ export function AboutModal(props: Props) {
         (response) => response.data
       ),
     staleTime: Infinity,
-    enabled: isSelfHosted(),
+    enabled:
```

Comment:
@beganovich @turbo124 The staging API helped me test this functionality, but the health_check endpoint is not available on the staging API. So, I made fix to avoid automatical logout.